test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -370,48 +366,6 @@ public void AddProductWithIdentity_MissingPrimaryColumn(SupportedLanguages lang)
             Assert.Equal(0, this.ExecuteScalar("SELECT COUNT(*) FROM dbo.ProductsWithMultiplePrimaryColumnsAndIdentity"));
         }
 
-        /// <summary>
-        /// Tests that when using a case sensitive database, an error is thrown if
-        /// the POCO fields case and column names case do not match.
-        /// </summary>
-        [Theory]
-        [SqlInlineData()]
-        // JSON serialization is case sensitive in Java
-        // TODO: https://github.com/Azure/azure-functions-sql-extension/issues/411
-        [UnsupportedLanguages(SupportedLanguages.Java)]
-        public void AddProductCaseSensitiveTest(SupportedLanguages lang)
-        {
-            // Set table info cache timeout to 0 minutes so that new collation gets picked up
-            var environmentVariables = new Dictionary<string, string>()
-            {
-                { "AZ_FUNC_TABLE_INFO_CACHE_TIMEOUT_MINUTES", "0" }
-            };
-            this.StartFunctionHost(nameof(AddProductParams), lang, false, null, environmentVariables);
-
-            // Change database collation to case sensitive
-            this.ExecuteNonQuery($"ALTER DATABASE {this.DatabaseName} SET Single_User WITH ROLLBACK IMMEDIATE; ALTER DATABASE {this.DatabaseName} COLLATE Latin1_General_CS_AS; ALTER DATABASE {this.DatabaseName} SET Multi_User;");
```

Comment:
We should still have tests that : 

1. Verify a case sensitive database can be used without error (at least one for each binding type)
2. Using incorrect casing for a column name throws an error

Reply:
I will open a separate PR to add the tests in all languages since this PR is quite large already